drydock/patches/openedx-common-settings

```diff
@@ -18,3 +18,6 @@ XBLOCK_SETTINGS["ScormXBlock"] = {
     "STORAGE_FUNC": scorm_xblock_storage,
 }
 {% endif %}
+
+LOGGING.get("loggers", {}).pop("tracking")
```

Comment:
Why removing the tracking logger?. Also, do it using bracket indexing, if there's an error is better for it to fail early.

Reply:
This removes the duplicated logger and the `tracking.log` file.